pypeit/reduce.py

```diff
@@ -287,6 +289,48 @@ def local_skysub_extract(self, sciimg, sciivar, tilts, waveimg, global_sky, rn2i
 
         return None, None, None, None, None
 
+
+    def flexure_correct(self, sobjs, basename):
+        """ Correct for flexure
+
+        Args:
+            sobjs: SpecObjs object
+            maskslits: ndarray
+
+        Returns:
```

Comment:
Put this in the description.  This means that the function doesn't return anything, so just remove the `Returns` block.

Reply:
removed